test/hotspot/gtest/gc/shared/test_memset_with_concurrent_readers.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2016, 2022, Oracle and/or its affiliates. All rights reserved.
```

So as to match the JDK 17u version of the patch.

Reply:
Updated in new commit.